exercises/clock/.meta/solutions/clock.rb

```diff
@@ -1,24 +1,23 @@
 module BookKeeping
-  VERSION = 2
+  VERSION = 3
 end
 
 # Clock without dates exercise
 class Clock
-  def self.at(*args)
-    Clock.new(*args)
-  end
 
-  def initialize(hours=0, minutes=0)
-    @internal = hours * 60 + minutes
+  attr_reader :internal
```

Comment:
Is it sensible to have a method called `internal` that can be accessed externally?
What does it represent? Why not call it that?

Reply:
I agree.  I just left it in from the last solution.

You think `total_minutes` would be a good would be good there.

Reply:
That would be fine. Definitely a better name than `internal`

Reply:
Thanks